.github/workflows/ios_main.yml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: macos-latest
+    strategy:
+        matrix:
+          destination: ['platform=iOS Simulator,OS=13.1,name=iPhone 11']
+          scheme: [OpenDocumentReader]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Checkout submodule
+        run: git submodule sync --recursive && git submodule update --init --recursive
+      - name: Install pods
+        run: pod install
+      - name: Run build.sh
+        run: bash build.sh
+      - name: Build and test
```

Comment:
i'm nitpicking, but this is just "Build", not "test"

Reply:
test was removed